### PR TITLE
bump block size and a few related const

### PR DESCRIPTION
**Pull request by zekun000**
## Motivation

Allow larger batches for throughput

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/main/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/diem/diem/tree/main/developers.diem.com, and link to your PR here.)

## If targeting a release branch, please fill the below out as well

 * Justification and breaking nature (who does it affect? validators, full nodes, tooling, operators, AOS, etc.)
 * Comprehensive test results that demonstrate the fix working and not breaking existing workflows.
 * Why we must have it for V1 launch.
 * What workarounds and alternative we have if we do not push the PR.

### CODE REVIEW

**Review by gregnazario — DISMISSED** [review]

File: config/src/config/network_config.rs [hunk 1]
```diff
@@ -41,7 +41,7 @@ pub const MAX_CONCURRENT_NETWORK_REQS: usize = 100;
 pub const MAX_CONNECTION_DELAY_MS: u64 = 60_000; /* 1 minute */
 pub const MAX_FULLNODE_OUTBOUND_CONNECTIONS: usize = 3;
 pub const MAX_INBOUND_CONNECTIONS: usize = 100;
-pub const MAX_FRAME_SIZE: usize = 8 * 1024 * 1024; /* 8 MiB */
+pub const MAX_FRAME_SIZE: usize = 16 * 1024 * 1024; /* 8 MiB */
```
Comment (JoshLind) [inline config/src/config/network_config.rs:44]: nit: update the comment next to it? 😄

**Review by JoshLind — COMMENTED** [review]
I'm happy with this change! Will wait to see if there's any concerns from the storage folks, but if it lands, I'll also update state sync to use these new constants. 

Note: we might also want to consider increasing the `MAX_NUM_EPOCH_ENDING_LEDGER_INFO` but I'll propose that in a different PR.

File: config/src/config/network_config.rs (hunk 1 above)
Comment (zekun000) [reply config/src/config/network_config.rs:44]: LOL

**Comment by zekun000** [comment]
every time touching those numbers, I'm increasingly concerned about requests fail because of it exceeds frame size limit

**Review by gregnazario — DISMISSED** [review]

**Pull request merged**